src/cpp/session/modules/SessionProfiler.R

```diff
@@ -174,6 +179,11 @@
          unlink(profileDir, recursive = TRUE)
       }
 
+      rsconnectDir <- file.path(resources$tempPath, "rsconnect", "documents", paste(filePrefix, ".html", sep = ""))
```

Comment:
It's a path to a directory that happens to be named `filename.html` saved under `rsconnect/documents/filename.html`. Not sure why rsconnect names directories this way, but the code look about right, I think.